test/core/filters/filter_test_test.cc

```diff
@@ -132,7 +132,7 @@ using AddServerInitialMetadataFilterTest =
 
 TEST_F(NoOpFilterTest, NoOp) {}
 
-TEST_F(NoOpFilterTest, MakeCall) {
+TEST_F(NoOpFilterTest, MakeCallPair) {
```

Comment:
This doesn't actually appear to be calling `MakeCallPair()`.  Was this perhaps an accidental search-and-replace change?

Reply:
yeah... good catch :)